views/users/new.tpl

```diff
@@ -1,8 +1,35 @@
-{{ if .flash }}
-  {{ .flash }}
+{{ range $flashType, $flashMessage := .flash }}
+  {{ if $flashMessage }}
+    <div class="alert alert--{{$flashType}}">
```

Comment:
Can we move alert to something like `partial`? I think alert will be used more than one places.